core/src/main/java/ai/timefold/solver/core/impl/score/director/InnerScoreDirectorFactory.java

```diff
@@ -29,56 +30,53 @@ public interface InnerScoreDirectorFactory<Solution_, Score_ extends Score<Score
     InnerScoreDirector<Solution_, Score_> buildScoreDirector();
```

Comment:
Is there documentation for the default value of `ConstraintMatchPolicy` used in `buildScoreDirector()`?

Reply:
The method got removed.